### PR TITLE
feat: complete strapi configuration and integrate with python

### PR DESCRIPTION
**Pull request by BURG3R5**

### CODE REVIEW

File: content/config/database.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import path from 'path';
+// import path from 'path';
+var path = require('path');
```
Comment (BURG3R5) [inline content/config/database.ts:2]: Bruh. Use `import`

**Comment by BURG3R5** [comment]
@NanoNish

File: content/config/database.ts (hunk 1 above)
Comment (NanoNish) [reply content/config/database.ts:2]: lol it didnt import as es5 module on my pc

**Comment by NanoNish** [comment]
@BURG3R5 checkout `database.ts` once for the SSL config. Do you know if we need to change it?
Also, tell me any changes to be made to tables.

**Pull request merged**